src/sql/parts/notebook/notebook.component.ts

```diff
@@ -526,6 +424,15 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		provider.bindTo(this.contextKeyService);
 	}
 
+	public async save(): Promise<boolean> {
+		try {
+			return this._notebookParams.input.save();
+		} catch (err) {
+			this.notificationService.error(localize('saveFailed', 'Failed to save notebook: {0}', notebookUtils.getErrorMessage(err)));
+			return false;
```

Comment:
should this be Promise.resolve(false)? Or else don't try / catch and just return to caller who can handle the rejection in whatever way they want?

Reply:
removed exception handling